plugins/inputs/teamspeak/teamspeak.go

```diff
@@ -0,0 +1,97 @@
+package teamspeak
+
+import (
+	"github.com/multiplay/go-ts3"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+	"strconv"
+)
+
+type Teamspeak struct {
+	Server   string
+	Username string
+	Password string
+	Vservers []int
+
+	client *ts3.Client
+	connected bool
+}
+
+func (ts *Teamspeak) Description() string {
+	return "Reads metrics from a Teamspeak 3 Server via ServerQuery"
+}
+
+const sampleConfig = `
+  ## Server address for Teamspeak 3 ServerQuery
+  # server = "127.0.0.1:10011"
+  ## Username for ServerQuery
+  # username = "serverqueryuser"
+  ## Password for ServerQuery
+  # password = "secret"
+  ## Array of virtual servers
+  # vservers = [1]
```

Comment:
Any values that are required to be changed should be uncommented.  If there are reasonable default leave them commented out but then make sure the default is set.

For example, I would leave the default server and vservers commented.  I usually set the default values in the init func.  [Here is an example in the docker input](https://github.com/influxdata/telegraf/blob/master/plugins/inputs/docker/docker.go#L66).

For username/password, probably have them uncommented.  You can see how this will look with `telegraf --usage teamspeak` (also handy for copying into the README).